Libraries/Image/RCTImageLoader.m

```diff
@@ -371,7 +371,18 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithTag:(NSString *)imageTag
     // Check for cached response before reloading
     // TODO: move URL cache out of RCTImageLoader into its own module
     NSCachedURLResponse *cachedResponse = [_URLCache cachedResponseForRequest:request];
-    if (cachedResponse) {
+
+    while (cachedResponse) {
+      if ([cachedResponse.response isKindOfClass: [NSHTTPURLResponse class]]) {
```

Comment:
Last style nits, I promise :)

There shouldn't be a space between the `:` and the argument, so `isKindOfClass:[NSHTTPURLResponse class]`